app/facades/tagging_spreadsheet_facade.rb

```diff
@@ -0,0 +1,21 @@
+class TaggingSpreadsheetFacade < SimpleDelegator
```

Comment:
Sorry, this came out of an offline conversation with Tijmen. This is a presenter, or decorator (or a facade). I avoided the word presenter because we use that in other places in the context of serialisation (building payloads). But I'm happy to change this name to presenter if that's clearer!